index.js

```diff
@@ -1,30 +1,51 @@
+const multiRet = ['push', 'pop', 'shift', 'unshift']
+const mutArrMethods = ['reverse', 'sort', 'splice', 'fill', 'copyWithin']
```

Comment:
Sure, it's a good idea! And I think this way it's OK to move these back to the method. Even if the interpreter is not smart enough and allocates 3 strings each time you wrap something into a `mewt` (I don't think its that dumb though) it's not as bad as having 3 arrays created each time :)